sources/plugins/scg/scs/scgfilewriter.cpp

```diff
@@ -0,0 +1,69 @@
+#include "scgfilewriter.h"
+#include "scgscene.h"
+#include "scgnode.h"
+#include "scgpair.h"
+#include "scgbus.h"
+#include "scgcontour.h"
+
+#include <QMessageBox>
+#include <QTextCodec>
+#include <QApplication>
+#include <QFileInfo>
+
+SCgFileWriter::SCgFileWriter() {}
+
+SCgFileWriter::~SCgFileWriter() {}
+
+bool SCgFileWriter::save(QString file_name, QObject *input)
+{
+    SCgScene *scene = qobject_cast<SCgScene*>(input);
+
+    QFile fileOut(file_name);
+    QFile layoutFileOut(getLayoutFileName(file_name));
+    if (!openFile(fileOut) || !openFile(layoutFileOut))
+        return false;
+
+    stream.setOutputFiles(&fileOut, &layoutFileOut);
+
+    auto items = scene->items();
+    QGraphicsItem *item;
+
+    QVector<SCgObject *> objs;
+    QVector<SCgObject *> surfaceObjs;
+    foreach (item, items) {
```

Comment:
```suggestion
    foreach (item, items) 
    {
```
Please correct codestyle in all files to be consistent